packages/forms-web-app/src/controllers/final-comment/documents-check.js

```diff
@@ -0,0 +1,43 @@
+const { VIEW } = require('../../lib/views');
+
+exports.getDocumentsCheck = async (req, res) => {
+	//setting hasDocuments here will likely not be necessary when the
+	//final comment object is complete and this page is integrated
+	//into the full final comment journey
+	let hasDocuments = null;
+	if (
```

Comment:
Just a suggestion, as this is fine, but you might be able to shorten this by using optional chaining, eg. 
`res.render(VIEW.FINAL_COMMENT.DOCUMENTS_CHECK, { req.session?.finalComment?.hasDocuments });`
Because the expression will evaluate to undefined if any of the props in the chain are undefined, I think this should work.

Reply:
this is a good idea, if it works I'll change the `comments-question` controller as well